packages/docusaurus-theme-classic/src/theme/DocCardList/index.tsx

```diff
@@ -15,11 +15,20 @@ export default function DocCardList({
 }: {
   items: PropSidebarItem[];
 }): JSX.Element {
+  const filteredItems = items.filter((item) => {
```

Comment:
not sure this test is so great as it potentially not filter categories that have no link

the rule for filtering (not rendering) an item should be clear and in a single place (here)

Reply:
Don't really understand, can you give concrete  use case for testing?